docs/guide/index.md

```diff
@@ -67,27 +67,81 @@ $ bun create vite
 
 :::
 
+
+You can use `.` to scaffold inside an existing directory. 
+
+::: code-group
+
+```bash [NPM]
+$ npm create vite@latest .
+```
+
+```bash [Yarn]
+$ yarn create vite .
+```
+
+```bash [PNPM]
+$ pnpm create vite .
+```
+
+```bash [Bun]
+$ bun create vite .
+```
+
+:::
+
+
 Then follow the prompts!
 
 You can also directly specify the project name and the template you want to use via additional command line options. For example, to scaffold a Vite + Vue project, run:
 
-```bash
+::: code-group
+
+```bash [NPM]
 # npm 7+, extra double-dash is needed:
 npm create vite@latest my-vue-app -- --template vue
+```
 
-# yarn
+```bash [Yarn]
 yarn create vite my-vue-app --template vue
+```
 
-# pnpm
+```bash [PNPM]
 pnpm create vite my-vue-app --template vue
+```
 
-# bun
+```bash [Bun]
 bun create vite my-vue-app --template vue
 ```
 
```

Comment:
Can we keep this change only and revert the rest? I don't think the others need to be added/changed.

Reply:
Okay. That sounds good. If I see a need for any further changes, I will make a new PR. (Please @ tag me in the future so that I can get a notification email from GitHub. Thank you!)

Reply:
Okay. That sounds good. If I see a need for any further changes, I will make a new PR.

Reply:
Please let me know if there is anything else I must do. Thank you.

Reply:
@BhanuVikram as @bluwy explained, we'd like to merge only the change to add the code-group here, and leave the rest as it was before.

Reply:
@patak-dev Do I need to do anything to accomplish that?

Reply:
Yes, do the changes as explained in my comment. Please update this PR to change it.